pytim/gitim.py

```diff
@@ -9,10 +9,7 @@
 from scipy.spatial import distance
 from pytim import utilities
 import pytim
-try:
-    from pytetgen import Delaunay
-except:
-    from scipy.spatial import Delaunay
+from pytetgen import Delaunay
```

Comment:
are you sure you updated the version? it looks like you're removing some new changes.

Reply:
I just pulled from your Github, and this was what I got. I think the travis check were failing because of this.

Reply:
Somehow, I deleted it acidentally... I hope its fixed now.